src/screens/Drafting/Draft.tsx

```diff
@@ -230,6 +237,38 @@ const DraftScreen = ({ navigation }: any): ReactElement => {
         </View>
       </ScrollView>
       {/* <Button title="Back" style={styles.button} onPress={() => navigation.navigate('Home')} /> */}
+      <Modal visible={successModalVisibile} transparent={true} animationType={'slide'}>
+        <View style={styles.centeredView}>
+          <View style={styles.modal}>
+            <Text style={styles.modalText}>
+              Congratulations! You've submitted a job posting for {modalJobText}.
+            </Text>
+
+            <StyledButton
+              text={'POST ANOTHER JOB'}
+              textStyle={{ color: '#CC433C' }}
+              buttonStyle={{ backgroundColor: 'white', width: '100%' }}
+              onPress={() => {
+                navigation.goBack();
```

Comment:
just checking — what is this `goBack()` used for?

Reply:
it gets rid of the previous job post screen before navigating to the new screen. I thought it would feel more intuitive and like the form is fully submitted. Ultimately a UX decision so I'll do what you and Ceci think is best